.testcoverage.yml

```diff
@@ -49,6 +49,14 @@ override:
     path: ^webhook/logpipeline$
   - threshold: 83
     path: ^internal/utils/logpipeline$
+  - threshold: 75
+    path: ^webhook/tracepipeline/v1alpha1$
+  - threshold: 75
+    path: ^webhook/tracepipeline/v1beta1$
+  - threshold: 80
```

Comment:
we already have a coverage of 87% for `webhook/logpipeline/v1alpha1` and 84% for `webhook/logpipeline/v1beta1`. Can you increase these thresholds?

Reply:
updated after refactoring